static-build/nemu/build-static-nemu.sh

```diff
@@ -15,6 +15,31 @@ source "${script_dir}/../../scripts/lib.sh"
 config_dir="${script_dir}/../../scripts/"
 nemu_tar="kata-nemu-static.tar.gz"
 
+if [ $# -ne 0 ];then
+       arch="$1"
+       echo $arch
+       if [ "$arch" = "aarch64" ];then
+	       echo aaaaaaaaaaaa
```

Comment:
Debug output? Same comment for the "`b`", "`c`" and "`1`" `echo`'s below :smile: